indexer/main.go

```diff
@@ -100,8 +101,15 @@ func main() {
 	log.Println("Started retrieving versions for go packages\n")
 
 	var goErrorPackageMap = make(map[string]*GoPackage)
+	cluster := gocql.NewCluster("gophr.dev")
+	cluster.ProtoVersion = 4
+	cluster.Keyspace = "excelsior"
+	cluster.Consistency = gocql.One
+	session, _ := cluster.CreateSession()
+	defer session.Close()
```

Comment:
Is it better to create one session here and query with that one open session? Or create a logging component that when you call `db.Query(....)` will create a new session and perform your query